src/features/helios/components/grubShop/GrubShop.tsx

```diff
@@ -1,195 +0,0 @@
-import React, { useContext } from "react";
-import { Modal } from "react-bootstrap";
-
-import stall from "assets/buildings/grub_shop.png";
-import goblinChef from "assets/npcs/goblin_chef.gif";
-import shadow from "assets/npcs/shadow.png";
-
-import { GRID_WIDTH_PX, PIXEL_SCALE } from "features/game/lib/constants";
-import { Context } from "features/game/GameProvider";
-import { useActor } from "@xstate/react";
-import { GrubShopModal } from "./components/GrubShopModal";
-import { ITEM_DETAILS } from "features/game/types/images";
-import { ConsumableName } from "features/game/types/consumables";
-import { MapPlacement } from "features/game/expansion/components/MapPlacement";
-import { setImageWidth } from "lib/images";
-import { SUNNYSIDE } from "assets/sunnyside";
-import { DeliveryModal } from "features/island/delivery/Delivery";
-import { NPC } from "features/island/bumpkin/components/NPC";
-import { NPC_WEARABLES } from "lib/npcs";
-
-export const GrubShop: React.FC = () => {
-  const { gameService } = useContext(Context);
-  const [
-    {
-      context: { state },
-    },
-  ] = useActor(gameService);
-
-  const [showModal, setShowModal] = React.useState(false);
-
-  const openModal = () => {
-    setShowModal(true);
-  };
-
-  const closeModal = () => {
-    setShowModal(false);
-  };
-
-  const isClosed = !state.grubShop || state.grubShop.closesAt < Date.now();
-
-  const ordersFulfilled = state.grubOrdersFulfilled ?? [];
-  let lastFulfilledItemName: ConsumableName | undefined;
-  if (ordersFulfilled.length > 0) {
-    lastFulfilledItemName = state.grubShop?.orders.find(
-      (order) => order.id === ordersFulfilled[ordersFulfilled.length - 1].id
-    )?.name;
-  }
-
-  const showDeliveries = Date.now() > new Date("2023-05-31").getTime();
-
-  return (
-    <>
-      <Modal centered show={showModal} onHide={closeModal}>
-        {showDeliveries ? (
```

Comment:
We are actually showing the new deliveries in the Grub Shop component.

I would suggesting renaming this component to `DeliveryStall` and removing the Grub Shop related content 👌